test/e2e/windows/memory_limits.go

```diff
@@ -96,7 +96,7 @@ func checkNodeAllocatableTest(ctx context.Context, f *framework.Framework) {
 	ginkgo.By(fmt.Sprintf("Checking stated allocatable memory %v against calculated allocatable memory %v", &nodeMem.allocatable, calculatedNodeAlloc))
 
 	// sanity check against stated allocatable
-	framework.ExpectEqual(calculatedNodeAlloc.Cmp(nodeMem.allocatable), 0)
+	gomega.Expect(calculatedNodeAlloc.Cmp(nodeMem.allocatable)).To(gomega.Equal(0))
```

Comment:
Comparing the pod phase should give relatively obvious failure messages because the phases are strings, therefore I didn't point out the lack of an assertion message earlier.

But here we get a failure about two integers, with no indication what the comparison is about. Can you add an explanation string?